app/controllers/catalog_controller.rb

```diff
@@ -1549,8 +1549,10 @@ def available_container_templates(manager_id)
 
   def available_job_templates(manager_id)
     @edit[:new][:available_templates] = []
-    all_job_templates = ExtManagementSystem.find_by(:id => manager_id).send('configuration_scripts').collect { |t| [t.name, t.id] }.sort
-    all_workflow_templates = ExtManagementSystem.find_by(:id => manager_id).send('configuration_workflows').collect { |t| [t.name, t.id] }.sort
+    all_templates = ExtManagementSystem.find_by(:id => manager_id).send('configuration_scripts').sort_by(&:name)
```

Comment:
minor: you can do `ExtManagementSystem.find(manager_id)`

Reply:
Will `ExtManagementSystem.find(manager_id).configuration_scripts.sort_by(&:name)` works?